.builds/openbsd.yml

```diff
@@ -0,0 +1,27 @@
+image: openbsd/6.5
+
+packages:
+- gmake
+- cmake
+- libtool
+- unzip-6.0p11
+- autoconf-2.69p2
+- automake-1.15.1
```

Comment:
is `ninja` available? it will speed up the build (`gmake` will still be needed for third-party bundled Makefiles).

Reply:
ninja is available, I'll add it

Reply:
(IIRC the instructions for OpenBSD won't use ninja even if it's installed -- the top-level makefile is busted on OpenBSD)

Reply:
the top-level Makefile is only a convenience for those who don't want to use CMake directly, we can ignore it here.